src/main/java/com/sourcegraph/cody/agent/WebviewPostMessageParams.kt

```diff
@@ -54,3 +56,26 @@ data class WebviewPostMessageParams(val id: String, val message: ExtensionMessag
 data class ConfigFeatures(
     val attribution: Boolean,
 )
+
+data class EnhancedContextContextT(val groups: List<ContextGroup>)
```

Comment:
there is smth wrong with this name, isn't it?

Reply:
This matches the name on the extension side, which has a TODO to improve the name, but for now keeping them in sync would be handy.